README.md

```diff
@@ -55,6 +59,15 @@ make clean
 
 Cleans the build directory.
 
+## Note
+
+Your are not dependent on the [Makefile](Makefile).\
+For example: Windows user can also execute the hugo CLI commands manually like:
```

Comment:
Why the line break and the `\`? Convention is to make a line break at or after 80 characters (not breaking a word).

Also: "user" -> "users".

And still: this is not really related to the operating system. Windows users can use `make` as well, when installed. It's just that `make` is available with most Unix setups.

Reply:
Ok, tried to improve this context ...
What do you think?